src/components/Register.jsx

```diff
@@ -2,13 +2,14 @@ import React, { useState } from 'react';
 import { Button, Segment, Form } from 'semantic-ui-react';
 import Authentication from '../modules/Authentication';
 import { CardElement, useStripe, useElements } from '@stripe/react-stripe-js';
-import { Link } from 'react-router-dom';
+import { Link, useLocation } from 'react-router-dom';
 import { useSelector } from 'react-redux';
 import AuthenticationMessage from './AuthenticationMessage';
 
 const Register = () => {
   const { subscriber } = useSelector((state) => state);
   const [loading, setLoading] = useState(false);
+  const location = useLocation();
```

Comment:
How does this work?

Reply:
We're sending a `state` with the `Link` which we can access through `useLocation()` -> location.state

Reply:
To display a message ONLY if visitor has been redirected